questions.md

```diff
@@ -4,26 +4,54 @@
 
 1. On the UCI cluster, the resource request "-pe openmp 64" refers to the number of processors requested.  Does that
    request mean that your commands will use multiple processors?
+   #### Using the command '-pe openmp 64' just reserves 64 processors for your job, it doesn't mean that your job will actually use that many processors. The number of processors used by your job is determined by the actual script and parameters within it.
 2. In general, how do you know how many processors, how much RAM, how many files would be required/needed/written by the
    jobs you are running on the cluster?
+   #### If you haven't run a job before you can run a test job with sample inputs along with the following line of code before your commands: 
+   <pre><code>/usr/bin/time -v</pre></code>
+   #### This will output CPU time, RAM, and files required/needed/written requirements for your command.
 3. In order to be a "good citizen", you need to have some idea of much RAM your job requires.  In particular, you need
    to know the "peak" (i.e., maximum) RAM required at any point during execution.  Show an example of the shell command
    that you would use on a Linux machine to measure run time and peak ram usage of an arbitrary command, where the time/peak RAM values are written to a file.
+   <pre><code>/usr/bin/time -v bedtools bamtobed -i Sample_File.bam > stats.txt </pre></code>
+   #### The peak RAM your job uses will show up as Maximum resident set size in your generated text file stats.txt.
 4. What are the units of your answer for number 3?
+   #### Kilobytes
 5. What are the bash commands for the following operations:
 
     * Checking that a file exists
+    <pre><code> find /directory/ -name file_name </pre></code>
+    #### If the file exists then you should get an output of /directory/file_name. If it doesn't exist you won't get an output
```

Comment:
I suggest using a conditional statement instead of find as it allows you to run commands or output to stderr or stdout
if [ -e txt.md ]
then
    echo "Exists"
else
    echo "Does not exist"
fi
Using only the option -s will work is it checks if the file exists and is not empty. a one liner would be [ -s test.txt ] && echo "if true say something" || "if false say or do something else." here we are just doing logical operators. so && does a test to see if both are true it returns true and || is an or statement and returns something if anything is true. this is a bit confusing since it should only return if it is untrue but if you do FALSE || "something" bash will assume "something" is a TRUE value since it exists. you could just do if statements as they are easier to read and more legible. Honestly I almost exclusively use if statements and only use the one liners if i'm being lazy.